bin/main.rb

```diff
@@ -0,0 +1,213 @@
+# rubocop:disable Style/RedundantSelf
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Style/GlobalVars
+# rubocop:disable Style/ConditionalAssignment
+# rubocop:disable Lint/RedundantCopDisableDirective
```

Comment:
You have disabled so many cops. Just as suggested by the previous TSE this should not be the case. Instead of disabling, try and fix the errors